src/ResourceManager/HDInsight/Commands.HDInsight/help/New-AzureRmHDInsightCluster.md

```diff
@@ -388,6 +391,19 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisksPerWorkerNode
+Gets or sets the number of disks for worker node role in the cluster.```yaml
```

Comment:
@metafractal the trailing ` ```yaml ` string should be on a line by itself; this is a bug in the `platyPS` module. See any other parameter in this markdown file for an example of how this should look.

Reply:
@cormacpayne Fixed this in latest commit.